### PR TITLE
[Low Risk Cleanup] Removes double semicolons

### PR DESCRIPTION
**Pull request by kamronbatman**
Removes double semicolons since they are not needed.

No specific testing since there are no functional changes. This is low risk.

### CODE REVIEW

File: src/Servers/Kestrel/Transport.Libuv/src/Microsoft.AspNetCore.Server.Kestrel.Transport.Libuv.csproj [hunk 1]
```diff
@@ -2,12 +2,13 @@
 
   <PropertyGroup>
     <Description>Libuv transport for the ASP.NET Core Kestrel cross-platform web server.</Description>
-    <TargetFramework>$(DefaultNetCoreTargetFramework)</TargetFramework>
+    <TargetFramework>netcoreapp3.0</TargetFramework>
```
Comment (kamronbatman) [inline src/Servers/Kestrel/Transport.Libuv/src/Microsoft.AspNetCore.Server.Kestrel.Transport.Libuv.csproj]: I am going to revert these changes, sorry. This was due to Visual Studio.

File: src/Servers/Kestrel/Transport.Libuv/src/Microsoft.AspNetCore.Server.Kestrel.Transport.Libuv.csproj [hunk 2]
```diff
@@ -2,12 +2,13 @@
...
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <PackageTags>aspnetcore;kestrel</PackageTags>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <NoWarn>CS1591;$(NoWarn)</NoWarn>
     <IsShippingPackage>true</IsShippingPackage>
+    <Nullable>enable</Nullable>
```
Comment (kamronbatman) [inline src/Servers/Kestrel/Transport.Libuv/src/Microsoft.AspNetCore.Server.Kestrel.Transport.Libuv.csproj]: Same as above.

**Comment by Tratcher** [comment]
@aspnet-hello triage https://dev.azure.com/dnceng/public/_build/results?buildId=414066

**Comment by aspnet-hello** [comment]
This comment was made automatically. If there is a problem contact aspnetcore-build@microsoft.com.

I've triaged the above build. I've created/commented on the following issue(s)
https://github.com/aspnet/AspNetCore-Internal/issues/3295

**Comment by kamronbatman** [comment]
The triage doesn't exist?

**Comment by BrennanConroy** [comment]
@kamronbatman don't worry about that, it's for our internal tracking!

The change looks good, I'll approve it and we'll merge it hopefully later today.

**Review by BrennanConroy — APPROVED** [review]

**Review by halter73 — APPROVED** [review]

**Comment by halter73** [comment]
Thanks @kamronbatman!
